linters/perlcritic/perlcritic.test.ts

```diff
@@ -0,0 +1,3 @@
+import { linterCheckTest } from "tests";
+
+linterCheckTest({ linterName: "perlcritic", namedTestPrefixes: ["basic"] });
```

Comment:
```suggestion
import { linterFmtTest } from "tests";
import { skipOS }  from "tests/utils";

linterCheckTest({ linterName: "perltidy", namedTestPrefixes: ["basic"], skipTestIf: skipOS(["darwin"]) });
```

Reply:
Not skipped if macOS is configured in GHA workflow.